Services/Certificate/classes/Placeholder/Values/class.ilDefaultPlaceholderValues.php

```diff
@@ -116,7 +116,6 @@ public function __construct(
 			'DATETIME'           => '',
 			'DATE_COMPLETED'     => '',
 			'DATETIME_COMPLETED' => '',
-			'CLIENT_WEB_DIR'     => ''
```

Comment:
@iszmais This change (only this line) should be also applied for `trunk`

Reply:
moved this to another branch to get a better overview.